livekit_models.proto

```diff
@@ -391,8 +392,124 @@ enum ReconnectReason {
   RR_SWITCH_CANDIDATE = 4;
 }
 
+enum SubscriptionStatus {
+  Unsubscribed = 0;
+  Desired = 1;
+  Subscribed = 2;
+}
+
 enum SubscriptionError {
   SE_UNKNOWN = 0;
   SE_CODEC_UNSUPPORTED = 1;
   SE_TRACK_NOTFOUND = 2;
 }
+
+message ClientReport {
+
+  message RTCSenderStats {
+    // ice-candidate-pair
+    uint32 available_outgoing_bitrate = 1;
+    // local-candidate
+    string networkType = 3;
+    // outbound-rtp
+    uint64 packetsSent = 4;
```

Comment:
would suggest `snake_case` for field names. 

though I'm not sure if network level stats would be helpful, since we do get these on the server side. For v1, let's start with minimal stats?

Reply:
missed snake case here. Will remove `packets_sent` entirely then.